Classes/Interfaces/Hook/RegistrationView.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace OliverKlee\Seminars\Interfaces\Hook;
+
+use OliverKlee\Seminars\Interfaces\Hook;
+
+/**
+ * This interface needs to be used for hooks concerning the registration view.
+ *
+ * @author Michael Kramer <m.kramer@mxp.de>
+ */
+interface RegistrationView extends Hook
```

Comment:
There is no registration view. Maybe we can call it `RegistrationForm`?

Reply:
Actually, yes, it is the registration form. In MVC terms this is hooking into the View part of the registration form. So using "View" is consistent with the other view types (what_to_display in FlexForms). Is RegistrationFormView OK?

Reply:
The `RegistrationForm` class is a bad mashup of a controller and a view (and more of a controller than a view). I'd prefer to call it `RegistrationForm` as long as the registration form has not been migrated to (i.e., rewritten in) Extbase/Fluid.